globus_sdk/transfer/client.py

```diff
@@ -1618,7 +1701,8 @@ def endpoint_manager_task_list(self, num_results=10, **params):
         status:
 
         >>> tc = TransferClient(...)
-        >>> for task in tc.endpoint_manager_task_list(num_results=None):
+        >>> for task in tc.endpoint_manager_task_list(
+        >>>     num_results=None, filter_status="ACTIVE"):
```

Comment:
I'm not sure what the desired style is here. This seemed to most closely resemble other examples.